src/Application/Queries/GetPaginatedRequisitions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Application.Paging;
+using Application.Repositories;
+using Domain.Enums;
+using Mapster;
+using MediatR;
+
+namespace Application.Queries
+{
+    public class GetPaginatedRequisitions
+    {
+        public record Query : PageRequest, IRequest<PaginatedList<RequisitionResponse>>
+        {
+            public DateTime? StartDate { get; set; }
+            public DateTime? EndDate { get; set; }
+            public string? ExpenseHead { get; set; }
+            public string? Department { get; set; }
+            public string? StatusFilter { get; set; } //Assumption is that FE sends comma separated values
```

Comment:
please remove assuptions in your implemetation.. you have to be solid on what the contract is and have frontend adhere to the contract

Reply:
Removed